### ISSUE TITLE
Call _ready when creating scene

### ISSUE THREAD

**Issue by myin142** [CodeDarigan/WAT#149]
Is it possible to call `_ready()` on all nodes when you create a scene with `direct.scene()`? Currently I have to manually get all nodes and call the `_ready()` function to initialize variables that I set with `onready var`.

**Comment by CodeDarigan** [comment]
_ready runs when you enter a Node into the tree. Did you try adding the node as a child?

**Comment by myin142** [comment]
How would I do that? I don't think I have seen anything like that in the Docs or in your Tests. I am using the `direct.scene` to create the director and then get the node as a double. Like in your `scene_director.test.gd`.

**Comment by CodeDarigan** [comment]
var director = direct.scene(<path>)
var double = director.double()
add_child(double) // _ready should run here

(I guess it might not be clear but Tests are added to the tree when being run).

**Comment by myin142** [comment]
I think the `_ready` is being called now. But variables that are initialized with `get_node` and a path seem to be null. For example:

```gdscript
export var player_path: NodePath
onready var player: Spatial = get_node(player_path)
```
It works when I set the `player_path` manually. Can this somehow be fixed?

**Comment by CodeDarigan** [comment]
I have a few ideas. I'll get back to you.

**Comment by CodeDarigan** [comment]
@myin142 This should be resolved by 4760fb45ac923e33128319d26640016660caf1e3. There is a slight chance it might not work with scenes inherited from other scenes but I think it should be fine now.

**Comment by myin142** [comment]
Great, it works now. Thanks for the help!
